### ISSUE TITLE
Add proposed Pseudocode and implement Training Predictions

### ISSUE THREAD

**Issue by jenntenem** [jenntenem/style-change#3]

**Comment by jenntenem** [comment]
# ---- Pseudocode
# Load the dataset
data = load_dataset()

# Split the data into training and testing sets
train_data, test_data = split_data(data)

# Train the model on the training data
model = train_model(train_data)

# Make predictions on the testing data
predictions = predict(model, test_data)

# Evaluate the accuracy of the predictions
accuracy = evaluate(predictions, test_data)

# Print the accuracy
print('Accuracy:', accuracy)

**Comment by jenntenem** [comment]
// ---- Implementation
// Import the required libraries
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
from sklearn.neighbors import KNeighborsClassifier
from sklearn.metrics import accuracy_score

// Load the dataset
iris = load_iris()
X, y = iris.data, iris.target

// Split the data into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)

// Train the model on the training data
model = KNeighborsClassifier(n_neighbors=3)
model.fit(X_train, y_train)

// Make predictions on the testing data
y_pred = model.predict(X_test)

// Evaluate the accuracy of the predictions
accuracy = accuracy_score(y_test, y_pred)

// Print the accuracy
print('Accuracy:', accuracy)
